niworkflows/anat/mni.py

```diff
@@ -96,6 +97,11 @@ def _run_interface(self, runtime):
 
             self._config_ants(ants_settings)
 
+            if not isdefined(self.inputs.initial_moving_transform):
```

Comment:
One comment comes up to my mind:

AffineInitializer can be run just once, regardless the parameter set that is then hooked up, if this call was done before https://github.com/chrisfilo/niworkflows/blob/579741a350739fbb8f75fd0dff4e4fc7fb1fd3e7/niworkflows/anat/mni.py#L95.